README.rst

```diff
@@ -50,6 +51,7 @@ Here it is in context:
     .. reviewer-meta::
         :written-on: 1974-05-19
         :proofread-on: 1974-06-20
+        :dust_days_limit: 25
```

Comment:
Prefixing with `dust_` makes sense as a global setting, we can drop it here though. I'd rename the parameter `proofread-frequency`, which seems clearer in this context

Reply:
We'll, we've got multiple names for this same mecanism so as the configuration or project name.
I would prefer to keep one naming for this.